src/main/java/net/fabricmc/loom/task/launch/GenerateDLIConfigTask.java

```diff
@@ -234,7 +234,7 @@ public void run() throws IOException {
 					.map(File::getAbsolutePath)
 					.collect(Collectors.joining(File.pathSeparator));
 
-			final String intermediateNs = IntermediaryNamespaces.intermediaryNamespace(platform).toString();
+			final String intermediateNs = IntermediaryNamespaces.intermediaryNamespace(platform, getExtension()).toString();
```

Comment:
You shouldn't access the extension in the `@TaskAction` since it needs to support the config cache (like all tasks in the future, but this one is used in the config cache integration test).